tests/hpc/slurm_master_db.pm

```diff
@@ -53,39 +212,29 @@ sub run {
     barrier_wait("SLURM_SLAVE_SERVICE_ENABLED");
 
     $self->check_nodes_availability();
-    assert_script_run("srun -w slave-node00 date");
 
-    ## TODO provide more comprehensive checks for slurm
-    # job scheduling
-    for (my $i = 0; $i < 20; $i++) {
-        assert_script_run("srun -w slave-node00 date");
-        assert_script_run("srun -w slave-node01 date");
-    }
+    my $file = 'tmpresults.xml';
+    assert_script_run("touch $file");
+    script_run("echo \"<testsuite name='HPC single tests'>\" | tee $file");
```

Comment:
if the file is already created, you don't need a pipe, have you tried redirection ">>"?

Reply:
ah sure. Overall this is a quick and hackish way. I would like to move this to parsers and have HPC specific parser. So I wanted to just make it work for now.

In any case  ">>" should work, so I'll change this.